decorator/load.go

```diff
@@ -30,6 +38,17 @@ func Load(cfg *packages.Config, patterns ...string) ([]*Package, error) {
 
 	dpkgs := map[*packages.Package]*Package{}
 
+	includeLocalPkg := false
```

Comment:
I don't think we need any changes to the `Load` function. It's a convenience function, so you can easily copy it and make tweaks.